papers/david_dotson/david_dotson.rst

```diff
@@ -512,7 +513,7 @@ Although it is possible to extend ``datreant`` objects with limbs to do complex
 ``datreant`` fundamentally serves as a Pythonic interface to the filesystem, bringing value to datasets and analysis results by making them easily accessible now and later.
 As data structures and file formats change, ``datreant`` objects can always be used in the same way to supplement the way these tools are used.
 
-Because a ``Treant`` is both a Python object and a filesystem object, they work remarkably well with distributed computation libraries such as dask.distributed_ [Roc15]_ and workflow execution frameworks such as Fireworks_ [Jai15]_.
+Because each Treant is both a Python object and a filesystem object, they work remarkably well with distributed computation libraries such as dask.distributed_ [Roc15]_ and workflow execution frameworks such as Fireworks_ [Jai15]_.
```

Comment:
I mostly made this to work smoothly with the "they" that follows, keeping with the theme of talking about Treants in the plural by this point. But the beginning of the sentence works best referring to a single Treant, so hence "each" instead of "a".

Does that hold up?